datadog_lambda/cold_start.py

```diff
@@ -50,12 +50,16 @@ def is_new_sandbox():
 
 def get_cold_start_tag():
     """Returns the cold start tag to be used in metrics"""
-    return "cold_start:{}".format(str(is_cold_start()).lower())
+    return "cold_start:true" if is_cold_start() else "cold_start:false"
```

Comment:
Can't we just?
```suggestion
    return f"cold_start:{is_cold_start()}"
```

Reply:
Sure, but why do all that work when we already know what the tag will be?

Using python's `timeit` module:

```bash
$ python -m timeit -s 'from datadog_lambda.cold_start import is_cold_start' '"cold_start:true" if is_cold_start() else "cold_start:false"'
10000000 loops, best of 5: 25.2 nsec per loop
$ python -m timeit -s 'from datadog_lambda.cold_start import is_cold_start' 'f"cold_start{is_cold_start()}"'
5000000 loops, best of 5: 86.4 nsec per loop

Reply:
Oh, I found an even faster way!!

```python
return "cold_start:true" if _cold_start else "cold_start:false"
```

Cuts it down to `24.7 nsec per loop` from `40.9 nsec per loop`!

Reply:
sounds good, thanks for the explanation